### PR TITLE
Adds in Syndicate Bombs

### PR DESCRIPTION
**Pull request by MaxIsJoe**
part of #6149

CL: [New] Adds in the syndicate bomb, It's a huge bomb that cannot be picked up and must be dragged/pushed around + can be wrenched down to make them immobile. They have a 90 second minimum time for detonation and are much stronger than X4s.

Improved code quality and separated explosive scripts into a base universal function/data script and interaction scripts. This is to avoid creating bugs when creating multiple interactions for bombs.

CL: [Fix] Fixed a bug related to the timer not displaying correctly on start when the explosive GUI first appears.

CL: [Fix] Fixed blurry GUI sprites with explosives. they're much more crispy clean now.

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
review pls

**Comment by MaxIsJoe** [comment]
and while we're at it; review  #8181 as well

**Review by Bod9001 — DISMISSED** [review]

File: UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs [hunk 1]
```diff
@@ -157,20 +71,25 @@ private bool CanAttatchToTarget(Matrix matrix, RegisterTile tile)
...
 		}
 
 		public void ServerPerformInteraction(PositionalHandApply interaction)
 		{
 			void Perform()
 			{
+				if (explosiveType == ExplosiveType.SyndicateBomb)
+				{
+					Logger.LogError(
+						$"{gameObject} IS NOT SUPPOSED TO BE ATTACHED TO OBJECTS NOR PICKED UP!!! - {pickupable.CanPickup}");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs]: Explain to me like I am dumb, why is this needed? Can't we simply not give AttachablExplosive nor Pickupable component to that bomb and that would be it?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs]: > Explain to me like I am dumb, why is this needed? Can't we simply not give AttachablExplosive nor Pickupable component to that bomb and that would be it?

Oh shoot, forgot to remove that.

File: UnityProject/Packages/packages-lock.json [hunk 2]
```diff
@@ -119,7 +119,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.scriptablebuildpipeline": {
-      "version": "1.15.2",
+      "version": "1.19.2",
```
Comment (Bod9001) [inline UnityProject/Packages/packages-lock.json]: is this accidental?

Comment (MaxIsJoe) [reply UnityProject/Packages/packages-lock.json]: yeah and fixed

**Review by Bod9001 — APPROVED** [review]
